### PR TITLE
Added Installation guide for Gitlab Pages

### PR DESCRIPTION
**Pull request by kolibril13**
As this might be useful for other people as well.

### CODE REVIEW

**Comment by kolibril13** [comment]
Closes #583

**Comment by mgeier** [comment]
Thanks for this PR!

There are a lot of unrelated changes, do you know why? Do you know how to avoid them?

**Comment by mgeier** [comment]
Can you please rebase on `master` in order to fix the build error?

File: doc/usage.ipynb [hunk 1]
```diff
@@ -876,7 +907,7 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3",
+   "display_name": "Python 3 (ipykernel)",
```
Comment (kolibril13) [inline doc/usage.ipynb:910]:
```suggestion
   "display_name": "Python 3",```

File: doc/usage.ipynb [hunk 2]
```diff
@@ -890,7 +921,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.9.2"
+   "version": "3.7.9"
```
Comment (kolibril13) [inline doc/usage.ipynb:924]:
```suggestion
   "version": "3.9.2"
```

File: doc/usage.ipynb [hunk 3]
```diff
@@ -876,7 +907,7 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3",
+   "display_name": "Python 3",```
```
Comment (kolibril13) [inline doc/usage.ipynb:910]:
```suggestion
   "display_name": "Python 3",
```

**Comment by kolibril13** [comment]
```
git rebase master
Current branch added-installation is up to date.
```
this is what I get.

**Comment by mgeier** [comment]
You should first `git pull` the latest `master` branch.

**Comment by kolibril13** [comment]
then I get this:
```
git pull
Already up to date.
```

**Comment by mgeier** [comment]
That's strange.

The branch of this PR is currently at 7334341, and its base (the parent of your first commit) is 4fb5586 (which at some point was the `master` branch of this repo).
In the meantime, the `master` branch has progressed to d75cf07.

What is the hash of your `master` branch?

Probably you have to pull from a different remote?

**Comment by kolibril13** [comment]
closed for #597

**Pull request closed without merging**